plugins/GSdx/res/tfx.fx

```diff
@@ -361,6 +370,11 @@ float4x4 sample_4p(float4 u)
 	return c;
 }
 
+float4 fetch_raw_color(float2 xy)
+{
+	RawTexture.Load(RawTextureSampler, xy);
```

Comment:
I'm getting runtime error crashes from this line.

Reply:
`float4 fetch_raw_color(int2 xy)`

This should be an int2, not float2.

Reply:
Still getting a crash tho.

Reply:
My bad, it needs an int3: RawTexture.Load(int3(xy, 0));